src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue

```diff
@@ -400,6 +424,16 @@ export default {
         cosignerAddress.value = n.cosignerList.length>0?getMultiSigCosigner.value.cosignerList[0].address:''
       }
     })
+
+    const findAccWithAddress = address =>{
+      return walletState.currentLoggedInWallet.accounts.find(acc=>acc.address==address)
+    }
+
+    const checkCosignBalance = computed(() => {
+      let cosignBalance = findAccWithAddress(cosignerAddress.value).balance;
+      return cosignBalance;
+    })
+
```

Comment:
same as above, findAccWithAddress can be null, return 0 if it's null

Reply:
noted, will revise it

Reply:
I have revised the code, please let me know if it done correctly. thanks

Reply:
if(findAcc..) is null, .balance will be undefined